neurolib/models/multimodel/builder/base/backend.py

```diff
@@ -220,6 +297,20 @@ def run(self, duration, dt, noise_input, **kwargs):
                 argdefs=(),
             )
         )
+        self.compiled_function = integrate
```

Comment:
save the function for later runs - remember - now compile once, run multiple times